lessons/meta/local-run/index.md

```diff
@@ -0,0 +1,277 @@
+- Položka můžeš být i něco jiného než text!
+```
+
+Druhým stavebním blokem je mapa (v Pythonu slovník), která se skládá z klíčů a hodnot, nezáleží v ní tedy na pořadí, protože se k informaci vždy člověk dostance pomocí klíče.
+Mapa se ve formátu YAML zapisuje následovně:
+
+```yaml
+klic1: Hodnota klíče klic1.
+klic2: 2
+klic3: I hodnoty v mapě mohou být něco jiného.
+klic4: |
+    Pokud potřebuješ zapsat nějaký delší text, uděláš to takhle.
+
+    Tento text budou dva separátní odstavce v rámci jednoho klíče.
+klic5:
+klic6: Jak vidíš v klíči klic5, hodnota může být i prázdá.
+```
+
+A pak samozřejmě jde i skládat hodnoty:
+
+```yaml
+klic1: Hodnota klíče
+klic2:
+- Tohle je pole, které patří pod klíč klic2
+- vnorenyklic: Můžeme skládat jednotlivé typy skoro do nekonečna.
+- vnorenyklic: Klíče musí být unikátní jen v rámci jedné mapy, takže to může být takto.
+klic3:
+  vnorenyklic: Mapa může obsahovat další mapu i takto.
+```
+
+### Povinné informace
+
+Teď když už se vyznáme ve formátu YAML, můžeme začít vytvářet soubor ``info.yml``.
+Celá definice kurzu je jedna velká mapa, kterou si postupně popíšeme a vyplníme.
+Pro potřeby kurzu je [zde]({{static("info.yml")}}) připravená šablona, kterou můžeš použít.
+
+Nejdřív prvních pár základních povinných údajů:
+
+* ``title`` slouží pro název kurz (nepovinně lze dodefinovat pomocí ``subtitle``)
+* ``description`` slouží pro krátký popis kurzu (který se zobrazí v seznamu kurzů)
+* ``long_description`` slouží pro dlouhý popis kurzu, který se zobrazí na stránce kurzu
+
+A teď už jen nepovinné údaje:
+
+* ``place`` slouží pro označení místa
+* ``time`` slouží pro informaci o času (není nutný žádný specifický formát)
+
+Jestli chceš, aby pro kurz šel vygenerovat iCal soubor s plánem lekcí, musíš dále poté vyplnit údaj ``default_time``.
+Tato hodnota musí být mapa, která obsahuje dva klíče ``start`` a ``end``, kde bude čas lekcí.
+Čas musí být ve formátu ``HH:MM`` a musí být obalen uvozovkami nebo apostrofy, takže třeba takhle:
+
+```yaml
+default_time:
+  start: '18:00'
+  end: '20:00'
+```
+
+Poslední nepovinná hodnota než se dostaneme k obsahu kurzu jsou proměné, které se definují klíčem ``vars`` a musí být také mapa.
+Proměné mohou upravovat obsah lekcí a stránek kurzu a pokud budeš vytvářet nebo upravovat materiály, můžeš si i definovat vlastní.
+
+První proměnou, kterou můžeš použít je ``coach-present``.
+Pokud je tvůj kurz s lektorem nebo koučem, nastav tuto proměnou na hodnotu ``true``.
+
+Druhou proměnou, kterou můžeš použít je ``user-gender``.
+Pokud víš, že na tvém kurzu budou lidé jen jednoho pohlaví, můžeš nastavit materiály (které tak byly napsány), aby vykreslily správně formátovaný obsah.
+Nastavíš to pomocí písmenka ``f`` pro ženy, ``m`` pro muže.
+Pokud proměnou nevyplníš, materiály vykreslí obě varianty.
+
+Další proměné, které můžeš použít jsou ``pyladies`` nebo ``mi-pyt``.
+Ty využij (nastavením na hodnotu ``true``), pokud odvozuješ svůj kurz od těchto základních kurzů – viz níže.
```

Comment:
Vždy, když odvozuješ? Co vlastně ty proměnné dělají?

Reply:
Ok, pravda, asi ne vždycky, to popíšu líp. Když odvozuješ a fakt děláš PyLadies, tak by to tam mělo být. Jinak je to jedno.

Proměná pyladies je primárně kvůli sjednocení složek na které text odkazuje, případně poznámky o handoutech. Proměná mi-pyt se sice používá na dvou místech, ale jsou to jen nějaký krátký komentáře...